libraries/AC_AutoTune/AC_AutoTune.cpp

```diff
@@ -240,24 +240,24 @@ void AC_AutoTune::run()
         return;
     }
 
-    float desired_yaw_rate_cds;  // used during manual control
-    get_pilot_desired_rp_yrate_cd(desired_roll_cd, desired_pitch_cd, desired_yaw_rate_cds);
-
+    float desired_yaw_rate_rads = 0.0;   // used during manual control
```

Comment:
I don't think we need to add the "= 0.0;" here because the line below will always set the variable to something